x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/columns.tsx

```diff
@@ -169,25 +167,30 @@ interface Props {
 }
 
 const ServiceNowColumn: React.FC<Props> = ({ theCase }) => {
-  const { hasDataToPush, isLoading } = useGetCaseUserActions(theCase.id);
-  const handleRenderDataToPush = useCallback(
-    () =>
-      isLoading ? (
-        <EuiLoadingSpinner />
-      ) : (
-        <p>
-          <EuiLink
-            data-test-subj={`case-table-column-external`}
-            href={theCase.externalService?.externalUrl}
-            target="_blank"
-          >
-            {theCase.externalService?.externalTitle}
-          </EuiLink>
-          {hasDataToPush ? i18n.REQUIRES_UPDATE : i18n.UP_TO_DATE}
-        </p>
-      ),
-    [hasDataToPush, isLoading, theCase.externalService]
-  );
+  const handleRenderDataToPush = useCallback(() => {
+    const lastCaseUpdate = theCase.updatedAt != null ? new Date(theCase.updatedAt) : null;
```

Comment:
this is the same logic as in `use_post_push_to_service.tsx`. can we lift it into a util file?

Reply:
it is not because one is using CaseUserActions and the other one is using the data form the external service